### PR TITLE
[ET-903] Add more context to the "Going" strings for the new RSVP box

### PR DESCRIPTION
**Pull request by andrasguseo**
[ET-903]

Added context to some strings and button labels.
Added article URLs to the same files pointing to the relevant knowledgebase articles.

[ET-903]: https://moderntribe.atlassian.net/browse/ET-903

### CODE REVIEW

File: src/views/v2/rsvp/actions/rsvp/going.php [hunk 1]
```diff
@@ -25,6 +25,6 @@ class="tribe-common-c-btn tribe-tickets__rsvp-actions-button-going tribe-common-
 		type="submit"
 		<?php tribe_disabled( $must_login ); ?>
 	>
-		<?php esc_html_e( 'Going', 'event-tickets' ); ?>
+		<?php echo esc_html_x( 'Going', 'Button label.','event-tickets' ); ?>
```
Comment (tr1b0t) [inline src/views/v2/rsvp/actions/rsvp/going.php:28]: :no_entry_sign: No space found after comma in argument list

<sub>hash: 70585eba371c202de942766c6a387246</sub>

File: src/views/v2/rsvp/details/attendance.php [hunk 2]
```diff
@@ -26,6 +26,6 @@
 		<?php echo esc_html( $rsvp->qty_sold ); ?>
 	</span>
 	<span class="tribe-tickets__rsvp-attendance-going tribe-common-h7 tribe-common-h--alt tribe-common-b3--min-medium">
-		<?php esc_html_e( 'Going', 'event-tickets' ); ?>
+		<?php echo esc_html_x( 'Going', 'Label below the attendance number.','event-tickets' ); ?>
```
Comment (tr1b0t) [inline src/views/v2/rsvp/details/attendance.php:29]: :no_entry_sign: No space found after comma in argument list

<sub>hash: e29ea854f396b6ab3bb9e3574723a0a9</sub>

File: src/views/blocks/rsvp/status/going.php [hunk 3]
```diff
@@ -22,7 +22,7 @@
 	class="tribe-block__rsvp__status-button tribe-block__rsvp__status-button--going<?php if ( 'yes' === $going ) { echo ' tribe-active'; }?>"
 	<?php echo disabled( 'yes', $going, false ); ?>
 	>
-		<span><?php esc_html_e( 'Going', 'event-tickets' ); ?></span>
+		<span><?php echo esc_html_x( 'Going', 'Button label.', 'event-tickets' ); ?></span>
```
Comment (paulmskim) [inline src/views/blocks/rsvp/status/going.php:25]: I would suggest more context here, maybe something like "Label for the RSVP going button"

File: src/views/blocks/rsvp/status/not-going.php [hunk 4]
```diff
@@ -41,7 +41,7 @@
 		class="tribe-block__rsvp__status-button tribe-block__rsvp__status-button--not-going<?php if ( 'no' === $going ) { echo ' tribe-active'; }?>"
 		<?php echo disabled( 'no', $going, false ); ?>
 	>
-		<span><?php esc_html_e( 'Not going', 'event-tickets' ); ?></span>
+		<span><?php echo esc_html_x( 'Not going', 'Button label.', 'event-tickets' ); ?></span>
```
Comment (paulmskim) [inline src/views/blocks/rsvp/status/not-going.php:44]: I would suggest more context here, maybe something like "Label for the RSVP not going button"

File: src/views/v2/rsvp/actions/rsvp/going.php [hunk 5]
```diff
@@ -25,6 +25,6 @@ class="tribe-common-c-btn tribe-tickets__rsvp-actions-button-going tribe-common-
 		type="submit"
 		<?php tribe_disabled( $must_login ); ?>
 	>
-		<?php esc_html_e( 'Going', 'event-tickets' ); ?>
+		<?php echo esc_html_x( 'Going', 'Button label.', 'event-tickets' ); ?>
```
Comment (paulmskim) [inline src/views/v2/rsvp/actions/rsvp/going.php:28]: same here

File: src/views/v2/rsvp/actions/rsvp/not-going.php [hunk 6]
```diff
@@ -40,6 +40,6 @@
 		class="tribe-common-cta tribe-common-cta--alt tribe-tickets__rsvp-actions-button-not-going"
 		<?php tribe_disabled( $must_login ); ?>
 	>
-		<?php esc_html_e( "Can't go", 'event-tickets' ); ?>
+		<?php echo esc_html_x( "Can't go", 'Button label.', 'event-tickets' ); ?>
```
Comment (paulmskim) [inline src/views/v2/rsvp/actions/rsvp/not-going.php:43]: same here

**Review by sc0ttkclark — CHANGES_REQUESTED** [review]
Needs more context and more cow bell

File: readme.txt [hunk 7]
```diff
@@ -121,6 +121,7 @@ Currently, the following add-ons are available for Event Tickets:
 = [5.0.1] TBD =
 
 * Fix - More thorough validity checking for post IDs, such as to account for a BuddyPress page having a Post ID of zero. [ET-899]
+* Tweak - We have added context to some of the strings and labels of the new RSVP block to allow more granular translation. [GTRIA-294]
```
Comment (sc0ttkclark) [inline readme.txt:124]:
```suggestion
* Tweak - We have added context to some of the strings and labels of the new RSVP block to allow more granular translation. [ET-903]
```

File: src/views/blocks/rsvp/status/not-going.php (hunk 4 above)
Comment (sc0ttkclark) [inline src/views/blocks/rsvp/status/not-going.php:44]:
```suggestion
		<span><?php echo esc_html_x( 'Not going', 'Label for the RSVP not going button', 'event-tickets' ); ?></span>
```

File: src/views/v2/rsvp/actions/rsvp/going.php (hunk 5 above)
Comment (sc0ttkclark) [reply src/views/v2/rsvp/actions/rsvp/going.php:28]:
```suggestion
		<?php echo esc_html_x( 'Going', 'Label for the RSVP going button', 'event-tickets' ); ?>
```

File: src/views/v2/rsvp/details/attendance.php [hunk 8]
```diff
@@ -26,6 +26,6 @@
 		<?php echo esc_html( $rsvp->qty_sold ); ?>
 	</span>
 	<span class="tribe-tickets__rsvp-attendance-going tribe-common-h7 tribe-common-h--alt tribe-common-b3--min-medium">
-		<?php esc_html_e( 'Going', 'event-tickets' ); ?>
+		<?php echo esc_html_x( 'Going', 'Label below the attendance number.', 'event-tickets' ); ?>
```
Comment (sc0ttkclark) [inline src/views/v2/rsvp/details/attendance.php:29]:
```suggestion
		<?php echo esc_html_x( 'Going', 'Label below the attendance number', 'event-tickets' ); ?>
```

File: src/views/v2/rsvp/actions/rsvp/not-going.php (hunk 6 above)
Comment (sc0ttkclark) [reply src/views/v2/rsvp/actions/rsvp/not-going.php:43]:
```suggestion
		<?php echo esc_html_x( "Can't go", 'Label for the RSVP "can't go" version of the not going button', 'event-tickets' ); ?>
```

**Review by sc0ttkclark — APPROVED** [review]

**Review by sc0ttkclark — DISMISSED** [review]

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 2 error(s)

File: src/views/v2/rsvp/actions/rsvp/not-going.php [hunk 9]
```diff
@@ -40,6 +40,6 @@
 		class="tribe-common-cta tribe-common-cta--alt tribe-tickets__rsvp-actions-button-not-going"
 		<?php tribe_disabled( $must_login ); ?>
 	>
-		<?php esc_html_e( "Can't go", 'event-tickets' ); ?>
+		<?php echo esc_html_x( "Can't go", 'Label for the RSVP "can't go" version of the not going button', 'event-tickets' ); ?>
```
Comment (tr1b0t) [inline src/views/v2/rsvp/actions/rsvp/not-going.php:43]: * :no_entry_sign: PHP syntax error: syntax error, unexpected 't' (T_STRING), expecting ',' or ')'
* :no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: 046be24cd71477823f1d6ca1853f41a2</sub>

Comment (paulmskim) [reply src/views/v2/rsvp/actions/rsvp/not-going.php:43]: @sc0ttkclark escape quote

Comment (sc0ttkclark) [reply src/views/v2/rsvp/actions/rsvp/not-going.php:43]:
```suggestion
		<?php echo esc_html_x( "Can't go", 'Label for the RSVP "can\'t go" version of the not going button', 'event-tickets' ); ?>
```

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**